.github/workflows/delete.packages.yml

```diff
@@ -0,0 +1,27 @@
+name: Delete old packages
+
+on: 
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'The name of the package to be deleted.'
+        required: true
+      num-old-versions-to-delete:
+        description: 'The number of old versions to delete starting from the oldest version.'
+        default: '1'
+        required: false
+
+jobs:
+  
+  delete:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Delete packages
+      id: deletePackages
+      uses: actions/delete-package-versions@v3
```

Comment:
Not a blocker for this PR, but could use `actions/delete-package-versions@v4` instead. There is an example in common repository.

Reply:
thanks, dependabot's PR just merged 🥳

Reply:
@monsieurleberre in order to use the new version of "delete-package-versions", it is required to add an extra parameter `package-type: 'nuget'`. This was already done in [common](https://github.com/trakx/common) repository.
<img width="711" alt="image" src="https://user-images.githubusercontent.com/6867927/216935684-67f003c6-62f2-4c18-9653-b32e1a5c0b10.png">

Reply:
OK Sorry I didn't see that, will do the change now! 🙏

Reply:
https://github.com/trakx/shrimpy-api-client/pull/114